js/script.js

```diff
@@ -64,22 +66,61 @@ function stickyHeaderInit() {
             TweenMax.to(header, 0.4, {opacity: 1});
         }
     });
+
+    var scrollTimeout;
+    var lastScrollTop = 0;
+    $(window).scroll(function () {
+        if (scrollTimeout) {
+            clearTimeout(scrollTimeout);
+            scrollTimeout = null;
+        }
+        scrollTimeout = setTimeout(scrollHandler, 50);
+    });
+
+    scrollHandler = function () {
+        var scrollTop = $(this).scrollTop();
+        var firstSectionBottom = $('.section--first').position().top + $(window).height();
```

Comment:
Tässä melkein käyttäisin pseudo-selektoria, "section--first" luokan voi kuka tahansa asettaa, mutta esim. section:first-of-type palauttaa aina ensimmäisen section sivulta.